src/module/index.js

```diff
@@ -0,0 +1,68 @@
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+const mytaskList = document.getElementById('myTasksList');
+const displayTask = () => {
+  mytaskList.innerHTML = '';
+  tasks.forEach((task) => {
+    const taskContainer = document.createElement('div');
+    taskContainer.classList = 'content';
+    taskContainer.index = `${task.index}`;
+    taskContainer.innerHTML = `<div class="taskInput" id='${task.index}'>
+                          <input type="checkbox"></input>
+                            <input id="info" class="${task.completed === true ? 'taskCmpleted' : 'edit'}"
+                              type="text" value="${task.description}">
+                            </input>
+                          </div>
+                          <i class="fa-solid fa-trash-can deleteTask" id="removeTask"></i>`;
+    const info = document.getElementById('info');
+    if (task.completed === true) {
+      info.classList.add('taskCompleted');
+    }
+    mytaskList.appendChild(taskContainer);
+  });
+};
+
+const newTask = document.getElementById('input');
+const addTask = (e) => {
+  if (((e instanceof KeyboardEvent) && (e.key === 'Enter')) || (e === 'clicked')) {
```

Comment:
- [ ] [ OPTIONAL]  If I type on enter key without any value on the input it is automatically saved on local storage. I would like to recommend preventing the fact that the input is not empty before saving the data on local storage 👍 

![local](https://github.com/Bhekisile/To-do-list/assets/6615394/67f45f54-dd9e-4172-8ef3-8dd64030db87)

Reply:
Thank you for the information and I appreciate all your help. Have a good rest of your day.